src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/New-AzureRmRecoveryServicesAsrFabric.md

```diff
@@ -27,10 +34,31 @@ The **New-AzureRmRecoveryServicesAsrFabric** cmdlet creates an Azure Site Recove
 PS C:\>  $currentJob = New-AzureRmRecoveryServicesAsrFabric -Name $FabricName
 ```
 
+### Example 2
+```
+PS C:\>  $currentJob = New-AzureRmRecoveryServicesAsrFabric -Azure -Name $fabricName -Location "eastus"
+PS C:\>  Get-ASRJob -name $job.id
+```
+
 Starts the fabric creation with passed name and returns the ASR job used to track the fabric creation operation.
```

Comment:
@viverm same comment about the description from the first example being moved to the new example. Also, it looks like the example you added won't work since the second line should be `Get-ASRJob -Name $currentJob.Id`

Reply:
done